.github/workflows/tests.yaml

```diff
@@ -36,4 +38,5 @@ jobs:
 
       - name: Python unit tests 
         run: |
-          pip install -e ".[dev]" && pytest -v python/fastsim/tests/
+          pip install -e ".[dev]" && pytest -v python/fastsim/tests/ 
+          pip install -e ".[dev]" && pytest -v python/fastsim/demos/
```

Comment:
I think this line can be reduced to just  
`pytest -v python/fastsim/demos/` 
@robinsteuteville , could you try that out?

Reply:
```suggestion
          pytest -v python/fastsim/demos/
```